paths.json

```diff
@@ -2,7 +2,8 @@
   "mappings": [
     "/content/piramalfinance-eds2/:/",
     "/content/piramalfinance-eds2/loan-key-features:/loan-key-features.json",
-    "/content/piramalfinance-eds2/metadata:/metadata.json"
+    "/content/piramalfinance-eds2/metadata:/metadata.json",
+    "/content/piramalfinance-eds2/metadata:/query-index.json"
```

Comment:
should be "/content/piramalfinance-eds2.resource/query-index.json:/query-index.json"